src/ScriptCs.Hosting/FileConsole.cs

```diff
@@ -35,9 +35,9 @@ public void WriteLine(string value)
             this.AppendLine(value);
         }
 
-        public string ReadLine()
+        public string ReadLine(string prompt)
```

Comment:
It is used.
On Mon, Nov 9, 2015 at 7:10 AM Filip W <notifications@github.com> wrote:

> In src/ScriptCs.Hosting/FileConsole.cs
> <https://github.com/scriptcs/scriptcs/pull/1119#discussion_r44284665>:
>
> > @@ -35,9 +35,9 @@ public void WriteLine(string value)
> >              this.AppendLine(value);
> >          }
> >
> > -        public string ReadLine()
> > +        public string ReadLine(string prompt)
>
> why is there prompt if it's not being used?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/1119/files#r44284665>.
>